addons/account/models/account_move.py

```diff
@@ -699,7 +699,23 @@ def _get_quick_edit_suggestions(self, amount_total, quick_edit_total_amount, mov
                 taxes = journal.company_id.account_sale_tax_id if journal.type == 'sale' else journal.company_id.account_purchase_tax_id
             taxes = self.env['account.fiscal.position'].browse(fiscal_position_id).map_tax(taxes)
         price_untaxed = taxes.with_context(force_price_include=True).compute_all(quick_edit_total_amount - amount_total)['total_excluded']
-        return {'account_id': account_id, 'tax_ids': taxes.ids, 'price_unit': price_untaxed}
+
+        invoice_date = date.today()
+        prev_move = self.search([('state', '=', 'posted'),
+                                 ('journal_id', '=', journal_id),
+                                 ('move_type', '=', move_type),
+                                 ('company_id', '=', journal.company_id.id),
+                                 ('invoice_date', '!=', False)],
+                                order='date desc, name desc', limit=1)
```

Comment:
Why the `name desc`?
You might not have to take the move_type into account if the sequence is shared between invoice/refund (check `refund_sequence`)

Reply:
To follow the same order as the default one of AccountMove (`_order = 'date desc, name desc, id desc'`) in case multiple account moves have the same date

Reply:
But you are only reading the `invoice_date`, so why is it not `invoice_date desc`?

Reply:
Because we want the last `account_move`, and by last I mean "the most recent one in the list view of the `account_move`' which uses the `date` and not the `invoice_date` as defined in `_order = 'date desc, name desc, id desc'

Reply:
Then just don't specify the order :D

Reply:
Oh yep, didn't know the default `_order` would be retrieved if we didn't specify it in the query ! :D